docs/concepts/clustering.rst

```diff
@@ -7,14 +7,14 @@ Data-Oriented Clustering
 Minimal Downtime Volume Migration
```

Comment:
:grey_question: This page seems optimistic to me. Or rather... like aspirational. Do we have a functioning ZFS backend right now? I thought that was sort of work-in progress.  Is there more work to clean up these pages?